eclipse-theia/theia
# Upgrade vscode-ripgrep's yarn.lock entry so we pick-up latest

#### What it does

Upgrade `vscode-ripgrep`'s yarn.lock entry so we pick-up latest

This will benefit the developers that build our example application behind a firewall, since `vscode-ripgrep` recently supports operating system environment proxy settings:
https://github.com/microsoft/vscode-ripgrep/pull/10

Extenders can upgrade their own application's `yarn.lock` since this change will not directly affect them

Fixes #8171

#### How to test

Build using `yarn` and confirm that `vscode-ripgrep` we pull is now `1.8.0`:

```
$ yarn why vscode-ripgrep
[...]
=> Found "vscode-ripgrep@1.8.0"

```

Test that `search-in-workspace` and `file-search` still work


#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
